funcs.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"html/template"
+	"os"
+	"path/filepath"
+)
+
+type FuncContext struct {
+	rootPath   string
+	themePath  string
+	publicPath string
+	currentCwd string
+	global     *GlobalConfig
+}
+
+func (ctx FuncContext) FuncMap() template.FuncMap {
+	return template.FuncMap{
+		"i18n":     ctx.I18n,
+		"readFile": ctx.ReadFile,
```

Comment:
It's better to use uppercase `ReadFile`, like other template variables?

Reply:
As referred to in #121,  [Hugo](https://gohugo.io/functions/readfile/) uses lowerCamelCase for all functions in `FuncMap`, and UpperCamelCase for all functions in tplData (e.g. `os.ReadFile`, `.Get`). I was following this naming style.